datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -1207,6 +1310,32 @@ public void validateDatastreamsUpdate(List<Datastream> datastreams) throws Datas
     }
   }
 
+  /**
+   * Validate if the partition assignment is supported for the datastream
+   * @param datastream datastream which needs the verification
+   * @throws DatastreamValidationException if partition assignment is not supported
+   */
+  public void validatePartitionAssignmentSupported(Datastream datastream) throws DatastreamValidationException {
```

Comment:
Isn't this also a function of assignment strategy?

Reply:
This verify if the partition is being managed by this connector, I will adjust the comment to avoid some confusions.